paella-matterhorn/javascript/05_initdelegate.js

```diff
@@ -58,22 +55,25 @@ function initPaellaMatterhorn(episodeId, onSuccess, onError) {
 					       paella.matterhorn.resourceId = "";
 					   }
 					   // end #DCE logging helper
-						
-						if (serie != undefined) {
-							asyncLoader.addCallback(new paella.JSONCallback({url:'/series/'+serie+'.json'}), "serie");
-							asyncLoader.addCallback(new paella.JSONCallback({url:'/series/'+serie+'/acl.json'}), "acl");
-						
-							asyncLoader.load(function() {
-									//Check for series
-									paella.matterhorn.serie = asyncLoader.getCallback("serie").data;
-									//Check for acl
-									paella.matterhorn.acl = asyncLoader.getCallback("acl").data;
-									if (onSuccess) onSuccess();
-								},
-								function() {
-									if (onError) onError();
-								}
-							);
+					   // start #DCE get series from serch endpoint (instead of series endpoint)
+					   if (serie != undefined) {
+					       base.ajax.get({
+					           url: '/search/series.json',
+					           params: {'id': serie }
+					           },
+					           function (data, contentType, code) {
+					           var jsonData = data;
+					           if (typeof (jsonData) == "string") jsonData = JSON.parse(jsonData);
+					           // #DCE verify that results returned at least one series
+					           var totalItems = parseInt(data[ 'search-results'].total);
+					           if (totalItems === 0) {
+					               showLoadErrorMessage(paella.dictionary.translate("No series found for series id") + ": \"" + serie + "\"");
+					               return;
+					           }
+					           searchSeriesToSeriesSeries(data[ 'search-results'].result);
```

Comment:
Yes, it's Ok not to set that. paraphrasing the comment in ticket MATT-1577:
The javascript segment "if (currentAce.action == "read") {this.permissions.canRead = true;}" is not accessed with pull #13. It gets skipped over because the "paella.matterhorn.acl" is no longer defined. When the "paella.matterhorn.acl " is not defined, the Paella UPV defaults to "this.permissions.canRead = true". This is what DCE wants because the Search Endpoint already filters out anonymous restricted recordings, and DCE  already filters out DCE access restricted recordings at this point. There is no need for the player to look at the series access control unless DCE decides to use the UPV soft editor feature instead of the Opensource ffmpeg editor.